### ISSUE TITLE
document mask_zero parameter of Embedding layer

### ISSUE THREAD

**Issue by pcyin** [fchollet/keras#364]
Currently the ```mask_zero``` parameter is not documented in http://keras.io/layers/embeddings/

**Comment by simonhughes22** [comment]
@pycin I've been wondering, what does this do? Prevent parts of the embedding vector from being updated?

**Comment by wxs** [comment]
Oh I've documented this now, in #346  I think you can close the issue.

@simonhughes22 the flag is for use in recurrent architectures where you might have variable length inputs. This flag tells the Embedding that you're giving it variable length input using 0 as a pad character. The Embedding then creates a binary mask that it propagates to subsequent layers.

Only some layers (mostly the recurrent ones) currently support being given a mask input along with their normal data input.
